comb_spec_searcher/specification.py

```diff
@@ -177,7 +177,9 @@ def count_objects_of_size(self, n: int, **parameters) -> int:
         Return the number of objects with the given parameters.
         Note, 'n' is reserved for the size of the object.
         """
-        return self.root_rule.count_objects_of_size(n, **parameters)
+        limit = n * self.number_of_rules()
```

Comment:
is there a good reasn for this value?

Reply:
My reasoning was that your tree has no more that `num_rule` level and you recur at most `n` times. Of course this is not super accurate and does not account for multiple variables

Reply:
It is easy to alter if we come into similar problems. This is ready to merge in my opinion.